web/src/components/njwds-extended/HorizontalStepper.tsx

```diff
@@ -9,6 +9,7 @@ interface Props {
   currentStep: number;
   onStepClicked: (step: number) => void;
   suppressRefocusBehavior?: boolean;
+  inlineDialog?: boolean;
```

Comment:
This is for an upcoming ticket to consolidate the general guidance environment tasks. It might change as I implement that ticket but this is just an initial stab at including the new statuses.